admin_app_2/src/components/NavBar.tsx

```diff
@@ -0,0 +1,216 @@
+            passHref
+            style={{
+              textDecoration: "none",
+              //disable pointer events to prevent the link from being clicked
+              pointerEvents: "none",
+            }}
+          >
+            <MenuItem
+              key={page.title}
+              onClick={() => setAnchorElNav(null)}
+              sx={{
+                color:
+                  pathname === page.path
+                    ? appColors.outline
+                    : appColors.primary,
+              }}
+            >
+              {page.title}
+            </MenuItem>
+          </Link>
+        ))}
+      </Menu>
+    </Box>
+  );
+};
+
+const LargeScreenNavMenu = () => {
+  const pathname = usePathname();
+  return (
+    <Box
+      sx={[
+        {
+          flexGrow: 1,
+          display: { xs: "none", md: "flex" },
+        },
+        appStyles.justifyContentFlexEnd,
+      ]}
+    >
+      {pages.map((page) => (
+        <Link
+          href={page.path}
+          key={page.title}
+          passHref
+          style={{
+            textDecoration: "none",
+            //disable pointer events to prevent the link from being clicked
+            pointerEvents: "none",
+          }}
+        >
+          <Typography
+            key={page.title}
+            sx={{
+              m: sizes.baseGap,
+              color:
+                pathname === page.path ? appColors.white : appColors.outline,
+            }}
+          >
+            {page.title}
+          </Typography>
+        </Link>
+      ))}
+
+      <Layout.Spacer horizontal multiplier={4} />
+    </Box>
+  );
+};
+
+const UserDropdown = () => {
+  const router = useRouter();
+  const [anchorElUser, setAnchorElUser] = React.useState<null | HTMLElement>(
+    null,
+  );
+
+  const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElUser(event.currentTarget);
+  };
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null);
+  };
+  return (
+    <Box>
+      <Tooltip title="Open settings">
+        <IconButton onClick={handleOpenUserMenu} sx={{ p: 0 }}>
+          <Avatar
+            alt="Full access"
+            sx={{ width: sizes.icons.medium, height: sizes.icons.medium }}
+          />
+        </IconButton>
+      </Tooltip>
+      <Menu
+        sx={{ mt: "45px" }}
+        id="menu-appbar"
+        anchorEl={anchorElUser}
+        anchorOrigin={{
+          vertical: "top",
+          horizontal: "right",
+        }}
+        keepMounted
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "right",
+        }}
+        open={Boolean(anchorElUser)}
+        onClose={() => setAnchorElUser(null)}
+      >
+        {settings.map((setting) => (
+          <MenuItem key={setting} onClick={handleCloseUserMenu}>
+            <Typography textAlign="center">{setting}</Typography>
+          </MenuItem>
+        ))}
+      </Menu>
+    </Box>
+  );
+};
+export default NavBar;
```

Comment:
Most of this doesn't do anything at the moment. Remove and replace with a link to the dummy root login page?

Reply:
We'll need to fix all of Nav bar. Let's leave it for now.